services/ui/ws/display_binding.h

```diff
@@ -10,6 +10,7 @@
 #include "base/macros.h"
 #include "mojo/public/cpp/bindings/binding.h"
 #include "services/ui/public/interfaces/window_tree_host.mojom.h"
+#include "services/ui/public/interfaces/window_tree.mojom.h"
```

Comment:
is this include really used here? Maybe better to add it to a .cc file of the ancestor? This would avoid unnecessary includes in other headers, which inherit this interface.